index.bs

```diff
@@ -475,7 +475,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. If |extension| is not supported by this client platform, then [=continue=].
 
     1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, return [=a promise rejected with=] a {{DOMException}} whose name is
+        algorithm returned an error, return [=a promise rejected with=] a
         "{{TypeError}}", and terminate this algorithm.
```

Comment:
I'm fine with your suggestion to terminate the algorithm and propagate the error returned from the client processing.  The main thing I was trying to achieve was to enable errors to be caught and cause the operation to fail at that time, rather than being silently ignored.